index.js

```diff
@@ -22,8 +22,16 @@ exports.hasScreenCapturePermission = () => {
 		return true;
 	}
 
-	const screenCapturePermission = require('./build/Release/screencapturepermissions');
-	const hasPermission = screenCapturePermission.hasPermissions();
+	let hasPermission;
+
+	if (macosVersion.isGreaterThanOrEqualTo('10.16')) {
```

Comment:
In the latest `macosVersion` version, I fixed the handling of macOS 11, so if you upgrade, you can use `11` here.

Reply:
Done